cartographer_ros/cartographer_ros/node_main.cc

```diff
@@ -56,8 +56,8 @@ std::tuple<NodeOptions, TrajectoryOptions> LoadOptions() {
 
 void Run() {
   const auto options = LoadOptions();
-  constexpr double kTfBufferCacheTimeInNs = 1e15; // 1 million seconds
-  tf2_ros::Buffer tf_buffer{::tf2::Duration(kTfBufferCacheTimeInNs)};
+  constexpr double kTfBufferCacheTimeInSeconds = 1e6;
```

Comment:
this converts it back to the [same as upstream](https://github.com/googlecartographer/cartographer_ros/blob/9d5b221ed41783b15c84bc90b71527194b8d9a49/cartographer_ros/cartographer_ros/node_main.cc#L55)